apps/src/templates/ProfessionalLearningApplyBanner.jsx

```diff
@@ -5,6 +5,7 @@ class ProfessionalLearningApplyBanner extends React.Component {
   static propTypes = {
     useSignUpText: PropTypes.bool,
     nominated: PropTypes.bool,
+    closingSoon: PropTypes.bool,
```

Comment:
I could just send the value of the flag through as a string, I actually chose to pass closingSoon as a bool since that's all I need to know in this component right now

Reply:
Okay I changed my mind, I think I will just send the flag right through 👍